paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporter.kt

```diff
@@ -128,9 +128,9 @@ internal class DefaultEventReporter @Inject internal constructor(
         )
     }
 
-    override fun onShowNewPaymentOptionForm() {
+    override fun onShowNewPaymentOptions() {
```

Comment:
renamed this function and the corresponding `PaymentSheetEvent`, because this analytic event is for mc_{complete/custom}_sheet_newpm_show. It's not actually form-specific, that was just the horizontal mode behavior.